ironic/drivers/modules/drac/raid.py

```diff
@@ -202,6 +228,59 @@ def delete_virtual_disk(node, virtual_disk):
         raise exception.DracOperationError(error=exc)
 
 
+def reset_raid_config(node, raid_controller):
+    """Delete all virtual disk and unassign all hotspares physical disk
+
+    :param node: an ironic node object.
+    :param raid_controller: id of the RAID controller.
+    :returns: a dictionary containing the is_commit_required needed key with a
```

Comment:
The dictionary will contain both is_commit_required and is_reboot_required.  Can you update the comment to include both?

Reply:
Done.